.infra/index.ts

```diff
@@ -44,17 +38,25 @@ deployApplicationSuite({
   serviceAccount,
   secrets: envVars,
   apps: [{
-    port: 3000,
-    env: [nodeOptions(memory)],
+    env: [nodeOptions(maxMemory)],
     minReplicas: 3,
     maxReplicas: 7,
-    limits,
+    limits: {
+      memory: `${maxMemory}Mi`,
+    },
+    requests: {
+      cpu: '128m',
+      memory: `${memory}Mi`
+    },
     readinessProbe: {
       httpGet: { path: '/ready', port: 'http' },
       initialDelaySeconds: 10,
     },
     metric: { type: 'memory_cpu', cpu: 80 },
+    ports: [{ containerPort: 3000, name: 'http' }],
+    servicePorts: [{ targetPort: 3000, port: 80, name: 'http' }],
     createService: true,
+    spot: { enabled: true }
```

Comment:
@idoshamun maybe we want to lower the spot weight on the scraper?

Reply:
whatever you feel like it in this case :)

Reply:
```suggestion
    spot: { enabled: true, weight: 50 }
```
Going safe with 50/50, and to monitor it